src/components/Modal/ModalFooter/ModalFooter.types.ts

```diff
@@ -0,0 +1,13 @@
+import {ReactNode} from 'react';
+
+export type ModalFooterProps = Omit<React.ComponentPropsWithRef<'footer'>, 'className' | 'children'> & {
+    /**
+     * Children of the Modal
```

Comment:
Can you specify the exact component name, as you did for `ModalHeader`?
```suggestion
     * Children of the ModalFooter
```